packages/config/src/ConfigUtils.ts

```diff
@@ -46,16 +46,25 @@ export function getActiveProfileName(profileType: string, cmdArguments: ICommand
         defaultProfileName || profileType;
 }
 
+/**
+ * Describe the solutions we suggest for credential manager errors.
+ * @param displayName Display name of the CLI or credential manager
+ * @returns Additional details to store on the error object
+ */
+export function secureErrorDetails(displayName: string): string {
```

Comment:
Maybe we want the credential manager to provide these details instead of hardcoding them. If we use something other  then keytar in the future, the information provided here might not be accurate.

Reply:
Two solutions that come to mind for this:
1. Add a condition to the else clause added in CliProfileManager above, that only provides these details `if (CredentialManager.manager instanceof DefaultCredentialManager === true)`. And create a GitHub issue suggesting that we provide different error details if we use something other than keytar in the future.
2. Add an optional `possibleSolutions` array to the AbstractCredentialManager class that can be defined by CredentialManager implementations. For DefaultCredentialManager, items 1 and 2 could be defined in this array.

Suggestions? I'm leaning towards the first one, but that's probably because I'm lazy 🙂

Reply:
I kind of like the second one because it doesn't sound terrible to implement. But that's just me. We could definitely do the first one though, it's up to you. I just didn't want to approve this and forget, because if we move to something else, perhaps it won't have prereqs on Linux, perhaps it won't rely on the OS, or perhaps something goes wrong with another OS.